api/azure/webhook.go

```diff
@@ -62,9 +61,8 @@ func processBuild(aeAPI shared.AppEngineAPI, azureAPI API, owner, repo, sender,
 	log := shared.GetLogger(aeAPI.Context())
 	log.Infof("Fetching %s", artifactsURL)
 
-	slowClient, cancel := aeAPI.GetSlowHTTPClient(time.Minute)
-	defer cancel()
-	resp, err := slowClient.Get(artifactsURL)
+	client := aeAPI.GetHTTPClient()
```

Comment:
Could you briefly explain the difference between GetHTTPClient and GetSlowHTTPClient, as well as what does defer cancel() do?

Reply:
The difference is the timeout. The default timeout is 5s, while `GetSlowHTTPClient` allows you to specify a timeout (and if you do, you'd need to call `cancel()` to destroy the timeout context when you're done). Here we are only fetching a small JSON from an API so we don't really need 1min timeout.